test/rules-file-includes.c

```diff
@@ -49,24 +45,43 @@ test_rules(struct xkb_context *ctx, struct test_data *data)
         fprintf(stderr, "Expecting: %s\t%s\t%s\t%s\n",
                 data->keycodes, data->types, data->compat, data->symbols);
 
-    if (!xkb_components_from_rules(ctx, &rmlvo, &kccgst, NULL)) {
-        fprintf(stderr, "Received : FAILURE\n");
-        return data->should_fail;
+    bool passed = true;
+    for (int k = 0; k < 2; k++) {
```

Comment:
Any particular reason to test the private API?

Reply:
Not that I remember now 😅. It may have been about the RMLVO sanitizing (the private does not do it), but I do not remember… Oh well